standards/naming-conventions.md

```diff
@@ -0,0 +1,83 @@
+# Naming Conventions
+
+As part of the NYPL Engineering team, proper naming delivers clarity and consistency amongst our code, applications, services, consumers and more. The following sections will detail examples on how we (NYPL Engineering), agree on a standard for naming.
+
+Establishing proper naming will help each developer by:
+- Reducing the effort needed to read and understand source code
+
+- Enabling code reviews to focus on more important issues than arguing over syntax and naming standards
+
+- Enabling code quality review tools to focus their reporting mainly on significant issues other than syntax and style preferences
+
```

Comment:
Since this standard might get pretty long as more people (hopefully) include more specific naming conventions, should we include an index at the top that jumps to the different sections?